addons/account/views/report_invoice.xml

```diff
@@ -198,7 +198,7 @@
 
                                             <!--Tax totals-->
                                             <t t-set="tax_totals" t-value="o.tax_totals"/>
-                                            <t t-call="account.document_tax_totals"/>
+                                            <t t-if="tax_totals" t-call="account.document_tax_totals"/>
```

Comment:
This scenario is strange, as these report templates only make sense for invoice documents.
So I'm debating whats better
1. Hide the totals completely, as you've done; or
2. set a default value `t-value="o.tax_totals or {}"` and use `.get(key)` in `document_tax_totals_template`

I'm inclining toward 2, because we wouldn't "remove" anything.